src/wallet/wallet.h

```diff
@@ -45,17 +45,18 @@ extern bool fWalletRbf;
 
 static const unsigned int DEFAULT_KEYPOOL_SIZE = 100;
 //! -paytxfee default
-static const CAmount DEFAULT_TRANSACTION_FEE = 0;
+static const CAmount DEFAULT_TRANSACTION_FEE = COIN;
 //! -fallbackfee default
-static const CAmount DEFAULT_FALLBACK_FEE = COIN;
+//mlumin: 5/2021 scaled minimum, this likely will have to change for fee reduction
+static const CAmount DEFAULT_FALLBACK_FEE = COIN/10;
 //! -mintxfee default
 static const CAmount DEFAULT_TRANSACTION_MINFEE = COIN;
 //! minimum recommended increment for BIP 125 replacement txs
-static const CAmount WALLET_INCREMENTAL_RELAY_FEE = COIN * 5;
+static const CAmount WALLET_INCREMENTAL_RELAY_FEE = COIN/10 * 5;
```

Comment:
What's the rationale behind this?

Reply:
This is based on my scaling when introducing 1.14 fees being too aggressive, 5 DOGE for a replace-by-fee is likely excessive.

Reply:
Okay. Can we separate that from this PR then, because that's a wallet reconfiguration rather than a "fees backend" change? Currently 1.14.3 uses that 5 DOGE RBF so ideally, this we'd document your scaling and the decision to change it.

Reply:
Leaving this open as an active reminder until #2415 is merged and this is removed here.

Reply:
That's in https://github.com/dogecoin/dogecoin/pull/2415 but seems to not pass stably, checking why.